applications/FluidDynamicsApplication/python_scripts/apply_mass_conservation_check_process.py

```diff
@@ -0,0 +1,44 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+import math
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyMassConservationCheckProcess(Model, settings["Parameters"])
+
+class ApplyMassConservationCheckProcess(KratosMultiphysics.Process):
```

Comment:
the python version of this process does almost nothing, consider returning the c-process directly => see the statistics-process in the fluid-app

Reply:
If you do not mind, I would like to keep it and use it for the file handling which I was supposed to remove from Cpp (see the following entries below). Do you think this to be ok?

Reply:
now that you added extra implementation this makes sense